### PR TITLE
EZP-25248: Be able to empty the Trash

### PR DESCRIPTION
**Pull request by yannickroger**
Link: https://jira.ez.no/browse/EZP-25248

## Description
This PR adds a menu on the right of the Trash with a button to empty it.

trash

## Tests
Manual and unit tests

### CODE REVIEW

**Comment by yannickroger** [comment]
Ready for reviews ( ping @dpobel @StephaneDiot )

File: Tests/js/views/assets/ez-trashview-tests.js [hunk 1]
```diff
@@ -34,8 +41,11 @@ YUI.add('ez-trashview-tests', function (Y) {
                 contentType: this.contentTypeMock
             }];
 
+            this.barMock = new RenderedView();
```
Comment (dpobel) [inline Tests/js/views/assets/ez-trashview-tests.js]: it's not really a mock, you'd better call it `this.barView`. Also, the `RenderedView` instance sets its `rendered` to true but this is not checked anywhere AFAICS.

File: Tests/js/views/assets/ez-trashview-tests.js [hunk 2]
```diff
@@ -64,12 +75,71 @@ YUI.add('ez-trashview-tests', function (Y) {
...
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+        },
+
+        "Should provide the TrashBarView as an attribute": function () {
+            var trashBar = this.view.get('trashBar');
+
+            Assert.areSame(
```
Comment (dpobel) [inline Tests/js/views/assets/ez-trashview-tests.js]: `Y.Assert.isInstanceOf` does exactly that :)

**Comment by dpobel** [comment]
besides inline comments, +1

**Comment by StephaneDiot** [comment]
+1

**Comment by dpobel** [comment]
I've just realized you forgot to add the changes to Resources/public/fonts/selection.json

**Pull request merged**